ml/v1/ml.proto

```diff
@@ -48,6 +48,18 @@ service MLService {
             get: "/api/ml/v1/mlservices/{id}"
         };
     }
+
+    // Update an existing MLServices resource. If it does not exist, this will create a new one.
+    // Pass the desired updated state of MLServices to this call.
+    // Required permissions:
+    // - ml.mlservices.update
+    rpc UpdateMLServices(arangodb.cloud.ml.v1.MLServices) returns (arangodb.cloud.ml.v1.MLServices) {
```

Comment:
arangodb.cloud.ml.v1.MLServices --> MLServices (just like we do for GetMLServices)

Reply:
Fixed, thanks!